index.html

```diff
@@ -1,10 +1,3 @@
-<!--
```

Comment:
@AhmedMugtaba something with your local environment keeps trying to remove this block, but we need it in the codebase. Maybe your editor is doing something funny? Also while this change is in here we can't properly test locally, unfortunately.